Fails to build on macOS

### Steps to reproduce

Configure and make.

### Obtained behavior

```
libao.c:260:27: error: aliases are not supported on darwin
    __attribute__ ((weak, alias("libao_plugin_get")));
                          ^
1 error generated.
```

### Expected behavior

Builds successfully.

### Distribution

macOS 13.5

### Version of Speech-dispatcher

0.11.5

--- a/src/audio/alsa.c
+++ b/src/audio/alsa.c
@@ -1033,7 +1033,11 @@ spd_audio_plugin_t *alsa_plugin_get(void)
 	return &alsa_functions;
 }
 
-spd_audio_plugin_t *SPD_AUDIO_PLUGIN_ENTRY(void)
-    __attribute__ ((weak, alias("alsa_plugin_get")));
+spd_audio_plugin_t *
+    __attribute__ ((weak))
+    SPD_AUDIO_PLUGIN_ENTRY(void)
+{
+	return &alsa_functions;
+}
 #undef MSG
 #undef ERR

--- a/src/audio/libao.c
+++ b/src/audio/libao.c
@@ -256,7 +256,11 @@ spd_audio_plugin_t *libao_plugin_get(void)
 	return &libao_functions;
 }
 
-spd_audio_plugin_t *SPD_AUDIO_PLUGIN_ENTRY(void)
-    __attribute__ ((weak, alias("libao_plugin_get")));
+spd_audio_plugin_t *
+    __attribute__ ((weak))
+    SPD_AUDIO_PLUGIN_ENTRY(void)
+{
+	return &libao_functions;
+}
 #undef MSG
 #undef ERR

--- a/src/audio/nas.c
+++ b/src/audio/nas.c
@@ -262,5 +262,9 @@ spd_audio_plugin_t *nas_plugin_get(void)
 	return &nas_functions;
 }
 
-spd_audio_plugin_t *SPD_AUDIO_PLUGIN_ENTRY(void)
-    __attribute__ ((weak, alias("nas_plugin_get")));
+spd_audio_plugin_t *
+    __attribute__ ((weak))
+    SPD_AUDIO_PLUGIN_ENTRY(void)
+{
+	return &nas_functions;
+}

--- a/src/audio/oss.c
+++ b/src/audio/oss.c
@@ -519,7 +519,11 @@ spd_audio_plugin_t *oss_plugin_get(void)
 	return &oss_functions;
 }
 
-spd_audio_plugin_t *SPD_AUDIO_PLUGIN_ENTRY(void)
-    __attribute__ ((weak, alias("oss_plugin_get")));
+spd_audio_plugin_t *
+    __attribute__ ((weak))
+    SPD_AUDIO_PLUGIN_ENTRY(void)
+{
+	return &oss_functions;
+}
 #undef MSG
 #undef ERR

--- a/src/audio/pulse.c
+++ b/src/audio/pulse.c
@@ -356,8 +356,12 @@ spd_audio_plugin_t *pulse_plugin_get(void)
 	return &pulse_functions;
 }
 
-spd_audio_plugin_t *SPD_AUDIO_PLUGIN_ENTRY(void)
-    __attribute__ ((weak, alias("pulse_plugin_get")));
+spd_audio_plugin_t *
+    __attribute__ ((weak))
+    SPD_AUDIO_PLUGIN_ENTRY(void)
+{
+	return &pulse_functions;
+}
 
 #undef MSG
 #undef ERR

